app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AddProductIngredientsFragment.java

```diff
@@ -193,6 +207,25 @@ private String getTracesName(String languageCode, String tag) {
         return tag;
     }
 
+    /**
+     * To enable fast addition mode
+     *
+     * @param is
+     */
+    private void enableFastAdditionMode(boolean is) {
```

Comment:
boolean `is` could be more descriptive like `isEnabled` . But it is upto to you.

Reply:
Thanks for the suggestion. `isEnabled` is surely more descriptive, I will update it.